src/main/java/org/reactome/release/downloaddirectory/ProtegeExporter.java

```diff
@@ -42,7 +42,8 @@
 	// The code in GKB::WebUtils always writes protege files to /tmp/ and it doesn't look like that is configurable,
 	// so we'll work in /tmp as well.
 	private static final String PROTEGE_ARCHIVE_PATH = "/tmp/protege_files.tar";
-	private static final String PROTEGE_FILES_DIR = PROTEGE_ARCHIVE_PATH.replace(".tar", "/");
+	static final String PROTEGE_FILES_DIR = PROTEGE_ARCHIVE_PATH.replace(".tar", "/");
```

Comment:
Why remove `private` scope?

Reply:
The value for `PROTEGE_FILES_DIR` is needed in the integration test which is in the same package, so I changed it from `private` to package-private.